.stylelintignore

```diff
@@ -1,2 +1,3 @@
 public/scss/_variables.scss
 public/scss/_variables_reference.scss
+node_modules
```

Comment:
Please make a comment about this change in the PR so I can include that with the merge commit message.